hydra-node/src/Hydra/Chain/Direct/Tx.hs

```diff
@@ -142,14 +143,9 @@ instance Arbitrary ClosedThreadOutput where
 hydraHeadV1AssetName :: AssetName
 hydraHeadV1AssetName = AssetName (fromBuiltin hydraHeadV1)
 
--- | The metadata label used for identifying Hydra protocol transactions. As
--- suggested by a friendly large language model: The number most commonly
--- associated with "Hydra" is 5, as in the mythological creature Hydra, which
--- had multiple heads, and the number 5 often symbolizes multiplicity or
--- diversity. However, there is no specific numerical association for Hydra
--- smaller than 10000 beyond this mythological reference.
+-- | The metadata label used for identifying Hydra protocol transactions.
 hydraMetadataLabel :: Word64
-hydraMetadataLabel = 55555
+hydraMetadataLabel = 5
```

Comment:
Must: I need closure to why this would help the issue we had with auxiliary data hashes?